com.unity.render-pipelines.universal/Shaders/CameraMotionVectors.shader

```diff
@@ -22,30 +22,40 @@ Shader "Hidden/kMotion/CameraMotionVectors"
             #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
             #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/DeclareDepthTexture.hlsl"
 
-        #if defined(USING_STEREO_MATRICES)
-        float4x4 _PrevViewProjMStereo[2];
-#define _PrevViewProjM _PrevViewProjMStereo[unity_StereoEyeIndex]
+#if defined(USING_STEREO_MATRICES)
+                float4x4 _PrevViewProjStereo[2];
+                float4x4 _ViewProjStereo[2];
+
+                #define  _PrevViewProjM  _PrevViewProjStereo[unity_StereoEyeIndex]
+                #define  _ViewProjM      _ViewProjStereo[unity_StereoEyeIndex]
 #else
-#define  _PrevViewProjM _PrevViewProjMatrix
+
+                #define  _PrevViewProjM _PrevViewProjMatrix
+                #define  _ViewProjM     _ViewProjMatrix
 #endif
```

Comment:
This doesn't feel nice for users to handle extra complexity for stereo/non stereo path in their shaders. 
 Can we add the defines to URP shader library similar to how we have here: https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl#L20-L32